packages/core/echo/echo-db/src/testing/echo-test-builder.ts

```diff
@@ -118,6 +118,7 @@ export class EchoTestPeer extends Resource {
     const db = client.constructDatabase({ spaceKey });
     await db.setSpaceRoot(rootUrl);
     await db.open();
+    this._ctx.onDispose(() => db.close());
```

Comment:
EchoClient already closes all child databases on close. Is there something broken that's caused you to add this?

Reply:
Ok, I did not know this